src/sentry/api/helpers/group_index/validators/group.py

```diff
@@ -38,6 +38,7 @@ class GroupValidator(serializers.Serializer):
     # in minutes, max of one week
     ignoreUserWindow = serializers.IntegerField(max_value=7 * 24 * 60)
     assignedTo = ActorField()
+    priority = serializers.CharField()
```

Comment:
We should probably make this a ChoiceField with low medium and high as options to get proper validation